src/python/TaskWorker/Actions/DBSDataDiscovery.py

```diff
@@ -145,12 +147,16 @@ def execute(self, *args, **kwargs):
                                 " and contact the experts if the error persists.\nError reason: %s" % str(ex))
         self.keepOnlyDisks(locationsMap)
         if not locationsMap:
+<<<<<<< HEAD
```

Comment:
I am surprised that Jenkins did not catch this, but surely it can't be merged !

Reply:
In a way it cathed this, see E0001 [here](https://cmssdt.cern.ch/dmwm-jenkins/view/All/job/DMWM-CRABServer-PR-test/123/artifact/artifacts/PullRequestReport.html).
Anyway as said in the first comment, this PR is not intended to me merged until the corresponding recurring action is pushed.